helm/pay-wallet-values-dev.yaml

```diff
@@ -81,6 +81,9 @@ microservice-chart:
     SCHEDULER_CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS: "0"
     SCHEDULER_CDC_SEND_RETRY_MAX_ATTEMPTS: "1"
     SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS: "1000"
+    SCHEDULER_REDIS_RESUME_KEYSPACE: "pay-wallet-scheduler"
+    SCHEDULER_REDIS_RESUME_FALLBACK_IN_MIN: "10"
```

Comment:
SCHEDULER_REDIS_RESUME_FALLBACK_IN_MIN is not needed.

Reply:
done here a963bf41949734565b131774a0da0809fc4bfc61